### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio.

### ISSUE THREAD

**Issue by bibek98231** [Sanjita90/Tour-Management-Project#15]
**Product**: Home Page/Index Page
**Attachment:** 
![5](https://user-images.githubusercontent.com/64543701/118317503-dd809980-b517-11eb-8fdb-9757a7d23b6d.JPG)

**Steps to reproduce:** 
1) Open visual studio code.
2) Click the right button and choose an option Open with a live server.
3) After that it redirects to the browser.
4) When the browser is open, Click right button and choose the option inspect.
5) Then a small window is open at the side of the page where you will have to click the double right arrow icon which is at right side of sources.
6) Then choose or select the option Progressive Web App and Click to Generate report.
7) Report regarding error and mistake will appear.

**Environment:** Opera Internet Browser
**Expected and Actual Result:** The background and foreground doesn't have sufficient contrast ratio. Fix the contrast ratio will enhance its color.

**Comment by dhungana-bibek** [comment]
I have solved this problem.
![Screenshot (101)_LI](https://user-images.githubusercontent.com/80630264/118350158-fec3a300-b574-11eb-8488-d449eda1f41b.jpg)
